app/helpers/application_helper.rb

```diff
@@ -127,11 +127,19 @@ def link_to_function_if_authorized(name, function, options = {}, html_options =
   end
 
   def display_delete_if_authorized(options = {}, html_options = {})
+    if authorized_for(options)
```

Comment:
nitpick, a guard clause is more clear   (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)

```ruby
return "" unless authorized_for(options)
display_delete(options, html_options)
```